app/views/api/v2/hosts/show.json.rabl

```diff
@@ -25,3 +25,16 @@ end
 child :config_groups do
   extends "api/v2/config_groups/main"
 end
+
+host_additional_tabs(@host).each do |id, tab|
+  if tab.is_a? String
+    node do |host|
+      partial "api/v2/#{tab}", :object => host
+    end
+  else
+    child tab => id.to_sym do |aspect|
+      class_name = tab.class.name.underscore
+      extends "api/v2/#{class_name.pluralize}/base"
```

Comment:
oh but  actually app/views//katello is a 'view path', so really in our case we likely want just the class base name (i.e.  MyAspect rather than  MyModule::MyAspect)